src/web/Clipboard.ts

```diff
@@ -54,12 +55,12 @@ export class Clipboard extends RX.Clipboard {
         return node;
     }
 
-    private static _copyNode(node: HTMLTextAreaElement) {
+    private static _copyNode(node: HTMLSpanElement) {
         const selection = getSelection();
+        var range = document.createRange();
```

Comment:
You will have to remove selection before creating new one.. 

const selection = getSelection();
selection.removeAllRanges();

const range = document.createRange();
range.selectNodeContents(node);
selection.addRange(range);

document.execCommand('copy');
selection.removeAllRanges();

Reply:
Thanks for your comment. I have updated PR accordingly.